pkg/option/resolver/resolver.go

```diff
@@ -179,17 +180,17 @@ func readNodeOverrides(ctx context.Context, client client.Clientset, nodeName st
 	}
 
 	// We allow overriding individual key-value pairs by annotating the Node object
-	// with io.cilium.config/K=V
+	// with config.cilium.io/K=V
 	out := map[string]string{}
 
 	read := func(in map[string]string) {
 		for k, v := range in {
-			if strings.HasPrefix(k, "io.cilium.config/") {
+			if strings.HasPrefix(k, annotation.ConfigPrefix) || strings.HasPrefix(k, "io.cilium.config/") {
```

Comment:
I wonder whether the alias should be kept, given that this is a recent feature, and it doesn't seem to be documented at the moment.

Reply:
Cc @squeed

Reply:
here is the docs for this feature (:disappear:)

https://github.com/cilium/cilium/pull/23100

Reply:
You don't need to keep the old prefix; it has not yet been released. But, you will need to backport it to v1.13 if so.

Also, you'll need to update any tests.

Reply:
Perfect, I've dropped the alias and marked the PR for backport to v1.13. Thanks!